### ISSUE TITLE
Training resume lr_scheduler

### ISSUE THREAD

**Issue by TalalWasim** [badripatro/simba#10]
Hi,

I had a quick question regarding resume training. Where exactly is the lr_scheduler handled for resuming to ensure resume from the correct learning rate.

**Comment by badripatro** [comment]
The lr_scheduler will start resume from the last epoch.
